tests/build.gradle

```diff
@@ -31,7 +31,7 @@ test.dependsOn([
     ':core:pythonAction:distDocker',
     ':core:python2Action:distDocker',
     ':core:javaAction:distDocker',
-    ':core:swift3Action:distDocker',
```

Comment:
deprecate means announce to folks to use the new 3.1.1 but 3.0 is still is supported until we feel is good to remove.
We should continue to support and have tests for both